components/esp_websocket_client/esp_websocket_client.c

```diff
@@ -122,6 +133,52 @@ static uint64_t _tick_get_ms(void)
     return esp_timer_get_time()/1000;
 }
 
+static esp_err_t esp_websocket_setup_rx_buf(esp_websocket_client_handle_t client)
```

Comment:
Check for a valid client in all of these functions.

Reply:
I think this is an internal function and the client must not be NULL. So the check could be ignored.